hazelcast-spring/pom.xml

```diff
@@ -176,6 +176,13 @@
                 </dependency>
                 <dependency>
                     <groupId>com.hazelcast</groupId>
+                    <artifactId>hazelcast-client</artifactId>
+                    <scope>test</scope>
+                    <version>${project.parent.version}</version>
+                    <classifier>tests</classifier>
+                </dependency>
```

Comment:
The changes in the `ClientTestUtil` and therefore this change won't be needed after injecting `HazelcastClientProxy` in test class.